user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementMain.java

```diff
@@ -119,6 +126,10 @@ public MeasurementMain(
     add(measurementTemplateListComponent);
     add(measurementTemplateDownload);
     add(measurementDetailsComponent);
+
+    measurementDetailsComponent.addListener(
+        selectionChangedEvent -> updateSelectedMeasurementInfo(selectionChangedEvent.getSource().getNumberOfSelectedMeasurements()));
```

Comment:
This will lead to a null exception if the user switches between the measurement views within a project or between projects:

https://github.com/qbicsoftware/data-manager-app/assets/29627977/609bcf4d-83db-4c2e-af66-b19f7bd7e33d

`java.lang.NullPointerException: Cannot invoke "com.vaadin.flow.component.tabs.Tab.getLabel()" because the return value of "com.vaadin.flow.component.tabs.TabSheet.getSelectedTab()" is null
	at life.qbic.datamanager.views.projects.project.measurements.MeasurementDetailsComponent.getSelectedTabName(MeasurementDetailsComponent.java:369)
	at life.qbic.datamanager.views.projects.project.measurements.MeasurementDetailsComponent.getNumberOfSelectedMeasurements(MeasurementDetailsComponent.java:315)
	at life.qbic.datamanager.views.projects.project.measurements.MeasurementMain.lambda$new$e391be34$1(MeasurementMain.java:131)
	at life.qbic.datamanager.views.projects.project.measurements.MeasurementDetailsComponent.lambda$updateSelectedMeasurementsInfo$3(MeasurementDetailsComponent.java:273)`